src/System.Management.Automation/resources/Metadata.resx

```diff
@@ -150,6 +150,9 @@
   <data name="ValidateCountMaxLengthFailure" xml:space="preserve">
     <value>The number of provided arguments, ({1}), exceeds the maximum number of allowed arguments ({0}). Provide fewer than {0} arguments, and then try the command again.</value>
   </data>
+  <data name="ValidateCountEqualLengthFailure" xml:space="preserve">
+    <value>The number of provided arguments, ({1}), is not equal the number of allowed arguments ({0}). Provide exactly {0} arguments, and then try the command again.</value>
```

Comment:
This is definitely better, but I'm finding all 3 error messages too wordy and slightly inaccurate. First, maybe we should change `arguments` to `values` because there is a single argument with multiple values.

Second, maybe 1 error message is sufficient if we change it to:

```
The parameter requires at least `{0}` value(s) and no more than `{1}` value(s) - `{2}` value(s) were provided.
```